### DISCUSSION TITLE
Should something be shown on the invoice if partial paid + expired?

### DISCUSSION THREAD

**Discussion by Transisto** [btcpayserver/btcpayserver#4240] [General]
Currently it looks like this if someone is missing a few sats of his payment.

![image](https://user-images.githubusercontent.com/388716/198171975-4927cb7d-95d3-4fdc-9caa-3afe14677e4b.png)

**Comment by dennisreimann** [comment]
Good point, we should work thisninto Checkout v2, @dstrukt. 

We can offer a way to show the payment details on the result page (paid or expired) like on the payment view.

**Reply by dennisreimann** [reply]
This gets addressed in #4261. Thanks for the feedback!
